tests/test-generators.py

```diff
@@ -130,9 +147,17 @@ def validate_except_cost(output1: str, output2: str):
         elif "recursion-pairs.txt" in g:
             limit = 4
             strict_limit = 4
-
-        compare_output(consensus.output, expected, "")
-        compare_output(mempool.output, expected_mempool, "STRICT")
+        if (
+            "aa-million-messages.txt" not in g
+            and "aa-four-million-messages.txt" not in g
```

Comment:
it seems like this would be simpler if it matched the same pattern of the other exceptions